packages/nvidia_nat_core/src/nat/front_ends/fastapi/routes/generate.py

```
@@ -0,0 +1,273 @@
+            headers={"Content-Type": "text/event-stream; charset=utf-8"},
+            content=runner.streaming_generator_raw(
+                payload,
+                request,
+                streaming=streaming,
+                result_type=result_type,
+                output_type=output_type,
+                filter_steps=filter_steps,
+            ),
+        )
+
+    async def post_stream(request: Request, payload: Any = Body(), filter_steps: str | None = None):
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)